tsconfig.json

```diff
@@ -2,6 +2,7 @@
 	"extends": "@sindresorhus/tsconfig",
 	"compilerOptions": {
 		"noEmit": true,
+		"skipLibCheck": false,
```

Comment:
I'm not sure why this option was turned off before, but it prevented us from getting real-time error messages in the IDE while we were coding(for `.d.ts` files).

Reply:
This has been addressed in https://github.com/sindresorhus/type-fest/pull/785

Reply:
done